hatchet_sdk/worker/worker.py

```diff
@@ -87,14 +107,16 @@ def register_workflow(self, workflow: WorkflowMeta):
             logger.error(e)
             sys.exit(1)
 
-        def create_action_function(action_func):
-            def action_function(context):
+        def create_action_function(
+            action_func: Callable[..., Any]
+        ) -> Callable[..., Any]:
+            def action_function(context: Any) -> Any:
                 return action_func(workflow, context)
 
             if asyncio.iscoroutinefunction(action_func):
-                action_function.is_coroutine = True
+                setattr(action_function, "is_coroutine", True)
             else:
-                action_function.is_coroutine = False
+                setattr(action_function, "is_coroutine", False)
```

Comment:
escape hatch for now - I think we can probably define another `Protocol` for the `action_func`, but will do that later